rpcs3/Emu/system_config.h

```diff
@@ -131,6 +132,7 @@ struct cfg_root : cfg::node
 		cfg::_bool enable_3d{ this, "Enable 3D", false };
 		cfg::_int<1, 8> consecutive_frames_to_draw{ this, "Consecutive Frames To Draw", 1, true};
 		cfg::_int<1, 8> consecutive_frames_to_skip{ this, "Consecutive Frames To Skip", 1, true};
+		cfg::_bool enable_shader_interpreter{ this, "Enable Shader Interpreter", true };
```

Comment:
What's this for if you already have a disabled mode?

Reply:
Old implementation removed in later commits.